docs/manage/CLI.md

```diff
@@ -670,21 +736,39 @@ viam.service.vision.v1.VisionService.GetClassificationsFromCamera
 |        command option     |       description      | positional arguments
 | ----------- | ----------- | ----------- |
 | `list`      | list all robots that the authenticated session has access to, filtered by organization and location.  | - |
+| `api-key`   |  work with an api-key for your robot | `create`(see [positional arguments: api-key](#positional-arguments-api-key)) |
 | `status`      | retrieve robot status for a specified robot  | - |
 | `logs`      | retrieve logs for a specified robot | - |
 | `part`      | manage a specified robot part  | `status`, `run`, `logs`, `shell` (see [positional arguments: part](#positional-arguments-part)) |
 | `--help`      | return help      | - |
 
+##### Positional arguments: `api-key`
+
+<!-- prettier-ignore -->
+| argument | description |
+| ----------- | ----------- | ----------- |
+| `create`     | create an API key for a specific robot |
+| `--help`      | return help |
+
+##### Named arguments
+
+<!-- prettier-ignore -->
+| argument | description | applicable commands | required |
+| ----------- | ----------- | ----------- | ----------- |
+| `--robot-id`      | the robot to create an API key for |`create` | true |
+| `--name`     |  the optional name of the API key    |`create` | false |
+| `--org-id`      |  the optional organization ID to attach the key to  |`create` | false |
```

Comment:
Same comments as above: Use `api-key` instead of `create`, and add a note (use the same copy!) to indicate that a name is auto-generated if not supplied.